src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.tsx

```diff
@@ -118,28 +136,25 @@ export const AccountTransactions: React.FC = () => {
       t('Outflow'),
       t('Inflow'),
     ];
-    const convertDataToArray = data.financialAccountEntries.entries.map(
-      (entry) => [
-        entry.entryDate
-          ? dateFormatShort(DateTime.fromISO(entry.entryDate), locale)
-          : 'No entry date',
-        entry.description ?? '',
-        entry.category?.name ?? entry.category?.code ?? '',
-        entry.type === FinancialAccountEntryTypeEnum.Debit
-          ? currencyFormat(
-              formatNumber(entry.amount),
-              entry.currency,
-              locale,
-            ).replace(/[\xA0\u2000-\u200B\uFEFF]/g, ' ')
-          : '',
-        entry.type === FinancialAccountEntryTypeEnum.Credit
-          ? currencyFormat(
-              formatNumber(entry.amount),
-              entry.currency,
-              locale,
-            ).replace(/[\xA0\u2000-\u200B\uFEFF]/g, ' ')
-          : '',
-      ],
+    const convertDataToArray = data.financialAccountEntries.entries.reduce(
+      (array, entry) => {
+        return [
+          ...array,
+          [
+            entry.entryDate
+              ? dateFormatShort(DateTime.fromISO(entry.entryDate), locale)
+              : 'No entry date',
+            entry.description ?? '',
+            entry.category?.name ?? entry.category?.code ?? '',
+            entry.type === FinancialAccountEntryTypeEnum.Debit
+              ? `${formatAmountForExport(entry.amount, true)}`
```

Comment:
We're intentionally removing the currency from these numbers, right? If different entries can have different curriencies is that a good idea? Or do we know that they all end up being the same currency in practice?

Reply:
I believe each financial account has its own currency. This came up as a recommendation from Helpscout since the old MPDX also removes the currency